### PR TITLE
Updates markdown-it package

### PR DESCRIPTION
**Pull request by emikhasyak-okta**
## Description:
- updates markdown-it package

### Resolves:

* [OKTA-615910](https://oktainc.atlassian.net/browse/OKTA-615910)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/configmid.md [hunk 1]
```diff
@@ -8,11 +8,12 @@
     signOutRedirectUri=<SIGN_OUT_REDIRECT_URI>
     ```
    | Placeholder               | Value                                                                                                                                                        |
-   |---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------| ----- |
+   |---------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (emikhasyak-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/android/configmid.md:11]: the extra column broke the table. In preview, the table should look the same as in prod
https://64cb2d239d67e0343c74514b--reverent-murdock-829d24.netlify.app/docs/guides/sign-into-mobile-app-redirect/android/main/#configure-your-app
https://developer.okta.com/docs/guides/sign-into-mobile-app-redirect/android/main/#configure-your-app

**Review by paulwallace-okta — APPROVED** [review]
LGTM. Nice catch!

**Pull request merged**